types/index.d.ts

```diff
@@ -12,7 +14,9 @@ export function init(swaggerPath: string, options?: ajvValidatorOptions): void;
  * Middleware that validates the request against the swagger
  * file, according to the request method and route
  */
-export function validate(req: object, res: object, next: Function): void;
+declare function validate(ctx: Context, next: Function): void; // koa
+declare function validate(req: Request, res: Response, next: Function): void; // express
+export { validate };
```

Comment:
Does that correctly export both overloaded versions?

Reply:
Yes